reporting-engine-worker/src/main/java/org/ihtsdo/termserver/scripting/reports/release/PackageComparisonReport.java

```diff
@@ -0,0 +1,199 @@
+		publishedFolder = run.getParamValue(PUBLISHED_FOLDER);
+
+		super.init(run);
+	}
+
+	public void postInit() throws TermServerScriptException {
+		String[] columnHeadings = new String[] {
+				"Previous Release, Current Release, Comparison Results"
+		};
+		String[] tabNames = new String[] {
+				"Comparison Summary"
+		};
+		super.postInit(tabNames, columnHeadings, false);
+	}
+	
+	@Override
+	public Job getJob() {
+		JobParameters params = new JobParameters()
+				.add(PREVIOUS_RELEASE).withType(JobParameter.Type.STRING)
+				.add(PREVIOUS_RELEASE_PACKAGE).withType(JobParameter.Type.STRING)
+				.add(CURRENT_RELEASE).withType(JobParameter.Type.STRING)
+				.add(CURRENT_RELEASE_PACKAGE).withType(JobParameter.Type.STRING)
+				.add(PUBLISHED_FOLDER).withType(JobParameter.Type.STRING)
+				.add(MODULES).withType(JobParameter.Type.STRING)
+				.add(REPORT_OUTPUT_TYPES).withType(JobParameter.Type.HIDDEN).withDefaultValue(ReportConfiguration.ReportOutputType.GOOGLE.name())
+				.add(REPORT_FORMAT_TYPE).withType(JobParameter.Type.HIDDEN).withDefaultValue(ReportConfiguration.ReportFormatType.CSV.name())
+				.build();
+				
+		return new Job()
+				.withCategory(new JobCategory(JobType.REPORT, JobCategory.RELEASE_STATS))
+				.withName("Package Comparison Report")
+				.withDescription("This report compares two packages (zip archives) using Unix scripts with output captured into usual Google Sheets")
+				.withParameters(params)
+				.withTag(INT)
+				.withTag(MS)
+				.withProductionStatus(ProductionStatus.PROD_READY)
+				.withExpectedDuration(40)
+				.build();
+	}
+
+	@Override
+	protected void loadProjectSnapshot(boolean fsnOnly) throws TermServerScriptException, InterruptedException, IOException {
+	}
+	
+	public void runJob() throws TermServerScriptException {
+		if (System.getProperty("os.name").toLowerCase().startsWith("windows")) {
+			throw new TermServerScriptException("Windows operating system detected. This report can only run on Linux or MacOS.");
+		}
+
+		String uploadFolder = String.join("_",
+						previousRelease,
+						extractDate(previousReleasePackage),
+						currentRelease,
+						extractDate(currentReleasePackage),
+						LocalDateTime.now().format(DateTimeFormatter.ofPattern("uuuu-MM-dd-HHmm")));
+
+		String[] scriptArguments = new String[]{
+				doubleQuote(previousRelease),
+				doubleQuote(previousReleasePackage),
+				doubleQuote(currentRelease),
+				doubleQuote(currentReleasePackage),
+				doubleQuote(publishedFolder),
+				doubleQuote(uploadFolder)
+		};
+
+		String command = "./" + SCRIPT_NAME + " " + String.join(" ", scriptArguments);
+
+		ProcessBuilder builder = new ProcessBuilder();
+		builder.redirectErrorStream(true);
+		builder.directory(new File("scripts"));
+		builder.command("sh", "-c", command);
+
+		try {
+			Process process = builder.start();
+
+			try (BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()))) {
+				reader.lines().forEach(line -> info(line));
```

Comment:
Something to think about for phase 2 here:   because the worker dies after an hour or so, taking the logs with it forever, I think you should write the output of the script to a 2nd tab on the google sheet.   Call it "logs" or "diagnostics" or something.